<launch>
    <arg name="config_path" default="$(find new_package)/config/default.yaml"/>

    <!-- C++-based Node -->
    <node pkg="new_package" type="new_package_node" name="new_package" output="screen">
        <param name="config_path" value="$(arg config_path)"/>
    </node>
</launch>
